### ISSUE TITLE
Tauri 2.0 support?

### ISSUE THREAD

**Issue by norude** [tauri-apps/tauri-invoke-http#24]
Basically this crate is not compatible with tauri 2.0
I tried to fix what I could in my fork, but I am not qualified enough to migrate this crate to 2.0 fully. 
Please help

**Comment by ahkohd** [comment]
@norude I'll have a look at this.

**Comment by norude** [comment]
I've tried again and this time I've ran into some bigger issues regarding the `invoke_key`.
In the tauri crate `invoke_key` is at most `pub(crate)` (as far as I found) and there's no way to get it inside this rust code and thus no way to format it into the custom `initialization_script`. At the JS side, in regular tauri v2, the key is transmitted through the code in the default `initialization_script`, but since we have to override it, it isn't transmitted at all. So we have absolutely no access to the invoke key 

In tauri v1 it worked like [this](https://github.com/tauri-apps/tauri/blob/bec484c54a863e8c07306f9b20d3b4e0c7fcbc05/core/tauri/scripts/ipc.js#L97)
